src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
+  const deprecatedMessage = deprecatedCard && plugin.deprecatedMessage;
+  const cardClasses = classnames('card', { [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false });
```

Comment:
Then you can use `isDeprecated` here instead of this ternary operator

Reply:
@phated This is great to reduce the code and easy for the understaning. But may I ask why [styles.pluginDeprecatedCard] has to be inside the array?

Reply:
@donghwipark that syntax is not for an array, it is a "Computed property name". You can learn about this at https://medium.com/front-end-weekly/javascript-object-creation-356e504173a8